### PR TITLE
Provider package Edge: Edge worker supports queue handling

### PR DESCRIPTION
**Pull request by AutomationDev85**
This PR adds the possibility to the Edge Worker to add or remove queues in the queue field in the DB and the worker gets this information and starts consuming tasks of queues or stops consuming task of queues. 

This makes future handling worker queues from main instance possible without restarting the worker on a different queue.

### CODE REVIEW

File: providers/tests/edge/cli/test_edge_command.py [hunk 1]
```diff
@@ -27,27 +27,20 @@
...
     _EdgeWorkerCli,
-    _get_sysinfo,
     _Job,
 )
 from airflow.providers.edge.models.edge_job import EdgeJob
 from airflow.providers.edge.models.edge_worker import EdgeWorker, EdgeWorkerState
 from airflow.utils.state import TaskInstanceState
 
-from tests_common.test_utils.config import conf_vars
+from dev.tests_common.test_utils.config import conf_vars
```
Comment (jscheffl) [inline providers/tests/edge/cli/test_edge_command.py]: This has moved yesterday, please adjust - else tests will fail.
```suggestion
from tests_common.test_utils.config import conf_vars
```

**Review by jscheffl — COMMENTED** [review]
Some static checks and using an import which (sorry for that) has been moved yesterday. If this is fixed and CI is green good to merge in my eyes.

**Comment by jscheffl** [comment]
As since yesterday we are on main with the edge provider, please also update the version number of the provider.

File: providers/src/airflow/providers/edge/CHANGELOG.rst [hunk 2]
```diff
@@ -27,6 +27,10 @@
 Changelog
 ---------
 
+0.2.0pre0
+.........
+
```
Comment (jscheffl) [inline providers/src/airflow/providers/edge/CHANGELOG.rst:32]: Can you add some details into changelog as well?

Comment (majorosdonat) [reply providers/src/airflow/providers/edge/CHANGELOG.rst:32]: Yes, I'll add some

**Review by jscheffl — APPROVED** [review]

**Pull request merged**